lib/reporters/base.js

```diff
@@ -105,7 +105,7 @@ if (process.platform === 'win32') {
  * @param {string} type
```

Comment:
As long as you're at it, we should remove the `MOCHA_COLORS` environment variable  
[here](https://github.com/mochajs/mocha/blob/a7b587dc142faede788d92a4c2cec3323b1568ff/lib/reporters/base.js#L46). Seems to be yet another anachronism used to "solve" the same problem as `FORCE_COLOR`.

Reply:
hmm, OK.

- [ ] remove `MOCHA_COLORS`